com.ibm.wala.util/src/com/ibm/wala/util/PlatformUtil.java

```diff
@@ -46,14 +50,29 @@ public static boolean onIKVM() {
    * @throws IllegalStateException if boot classpath cannot be found
    */
   public static String[] getBootClassPathJars() {
-    String classpath = System.getProperty("sun.boot.class.path");
+    String classpath = null;
+    String javaVersion = System.getProperty("java.specification.version");
+    if (javaVersion.equals("9") || javaVersion.equals("10") || javaVersion.equals("11")) {
+      // java11 support for jmod files
+      try {
+        classpath =
+            String.join(
+                File.pathSeparator,
+                Files.list(Paths.get(System.getenv("JAVA_HOME"), "jmods"))
+                    .map(x -> x.toString())
```

Comment:
Prefer method reference instead of lambda: `.map(Path::toString)`

Reply:
Fixed